src/ias_proxy/src/lru_cache/mod.rs

```diff
@@ -0,0 +1,74 @@
+/*
+ Copyright 2018 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+------------------------------------------------------------------------------
+*/
+
+/// module LruCache, for use by IAS proxy
+use std::collections::{VecDeque};
+use std::collections::{HashMap};
+// use std::sync::{Mutex};
+use core::borrow::BorrowMut;
+
+#[derive(Debug, Clone)]
+pub struct LruCache {
+    // Size of the LRU cache
+    max_size: usize,
+    // A list to note which key is accessed first, it should be locked before accessing
+    // order: Mutex<VecDeque<String>>,
```

Comment:
Do away with dead code.

Reply:
Yes, I have cleaned up these unused code and improved error handling (not using panic) in unit test patch.